docs/build-apps.md

```diff
@@ -713,20 +748,197 @@ client.pushCardBundleInConversationThread(carousel)
+
+*Example*
+
+<pre class="prettyprint">
+const applePayPaymentRequest: ApplePayPaymentRequestType = {
+    requestIdentifier: "83f86edb-XXXXX", // UUID
+    payment: {
+        currencyCode: "USD",
+        lineItems: [
+            {
+                amount: "45",
+                label: "Earpods",
+                type: "final"
+            },
+            {
+                amount: "955",
+                label: "iPhone 12 mini",
+                type: "final"
+            }
+        ],
+        requiredBillingContactFields: ["email"],
+        requiredShippingContactFields: ["email"],
+        shippingMethods: [
+            {
+                amount: "10",
+                detail: "Available within an hour",
+                identifier: "in_store_pickup",
+                label: "In-StorePickup"
+            }
+        ],
+        total: {
+            amount: "1000",
+            label: "TOTAL",
+            type: "final"
+        }
+    },
+    receivedMessage: {
+        type: "CARD",
+        data: {
+            title: "Please check this payment request",
+            text: "Check this payment request and choose your shipping method",
+            actions: [],
+            style: "icon"
+        }
+    }
+}
+
+client.pushApplePayPaymentRequestInConversationThread(applePayPaymentRequest)
+.then(() => /* success apple pay payment request */)
+.catch((error: ActionError) =>
+    /* error.message -> Error on command request */
+    /* error.details (if exists) -> More details about the error if it exists */
+)
+</pre>
+
+*Result*
+
+- *Message for an operator in the conversation thread when CPA push an Apple Pay Payment Request:*
+![applePaymentRequest](./assets/images/cpa-apple-payment-request.png)
+
+
+- *Success payment:*
+![applePaymentSuccess](./assets/images/cpa-apple-payment-success.png)
+
+- *Failed payment:*
+![applePaymentFailed](./assets/images/cpa-apple-payment-failed.png)
+
+***
+
 
 ##### Style sheet
 The library also provides a standalone stylesheet with CSS variables built to fit iAdvize's design guidelines.
 
 An app can include it either in its HTML: 
 
 <pre class="prettyprint lang-html">
-&lt;link rel="stylesheet" src="https://static.iadvize.com/conversation-panel-app-lib/2.4.0/idzcpa.base.css"&gt;
+&lt;link rel="stylesheet" src="https://static.iadvize.com/conversation-panel-app-lib/2.9.0/idzcpa.base.css"&gt;
 </pre>
 
 Or as a top-level import inside a preprocessed-stylesheet: 
 
 <pre class="prettyprint lang-css">
-@import 'https://static.iadvize.com/conversation-panel-app-lib/2.4.0/idzcpa.base.css';
+@import 'https://static.iadvize.com/conversation-panel-app-lib/2.9.0/idzcpa.base.css';
 </pre>
 
 A complete description of the provided variables can be found in [our knowledge base](https://help.iadvize.com/hc/en-gb/articles/4404351307026-Conversation-Panel-Apps-Guidelines#5-how-to-easily-style-your-conversation-panel-app-for-a-consistent-user-interface-integration-in-the-desk).
```

Comment:
Le lien n'est plus bon :
https://help.iadvize.com/hc/en-gb/articles/4404351307026-Conversation-Panel-Apps-Guidelines#5-how-to-easily-style-your-custom-app-for-a-consistent-user-interface-integration-in-the-desk

Reply:
d'ailleurs, il faudra aussi mettre à jour cette page sur le help.iAdvize.com qui a des liens vers la version 1.2.0 ... cc @fadwanassiri